cmd/web/config/config_test.go

```diff
@@ -188,16 +188,28 @@ func TestLogFormat_UnmarshalText(t *testing.T) {
 	tests := []struct {
 		name    string
 		vt      LogFormat
+		want    LogFormat
 		args    args
 		wantErr bool
 	}{
-		// TODO: Add test cases.
+		{
```

Comment:
Excellent! But don't forget the erroneous case! Especially this implementation has an error condition worth testing